.github/workflows/integration-tests.yml

```diff
@@ -16,58 +16,91 @@ jobs:
       run:
         working-directory: .ansible/collections/ansible_collections/linode/cloud
     steps:
-      - name: checkout repo
+      - name: Checkout repo
         uses: actions/checkout@v3
         with:
           path: .ansible/collections/ansible_collections/linode/cloud
           fetch-depth: 0
           submodules: 'recursive'
 
-      - name: update packages
+      - name: Update packages
         run: sudo apt-get update -y
```

Comment:
```suggestion
```

I think we won't need `apt update` because we already removed the`apt install` command and we won't use `apt` anywhere else in this workflow?